.drone.starlark

```diff
@@ -1906,6 +1906,7 @@ def ldapService(ldapNeeded):
 				'LDAP_ORGANISATION': 'owncloud',
 				'LDAP_ADMIN_PASSWORD': 'admin',
 				'LDAP_TLS_VERIFY_CLIENT': 'never',
+				'HOSTNAME': 'ldap',
```

Comment:
Note: other core app drone has `HOSTNAME` defined here. So this is being made consistent with other CI.

Reply:
didn't work - reverted.